### PR TITLE
Add transform-box, add missing transform properties

### PR DESCRIPTION
**Pull request by jamesnw**

### CODE REVIEW

File: features/individual-transforms.yml [hunk 1]
```diff
@@ -2,3 +2,10 @@ name: Individual transform properties
 description: Transform elements with separate `translate`, `rotate`, and `scale` CSS properties.
```
Comment (autonome) [inline features/individual-transforms.yml:2]: non-blocking suggestion: The description could be more explicit about how these allow individual transformation vs the grouped property, giving context to why the feature is named as such. Eg, I like the "simple transforms independently" in the spec language.

File: features/transform-box.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: transform-box
+description: The `transform-box` CSS property sets the reference box for transformations.
```
Comment (autonome) [inline features/transform-box.yml:2]: Ditto, I like the spec language, which gives just a bit more colour: "relative to the position and dimensions of the reference box". (The MDN description I found oddly phrased.)

**Review by autonome — COMMENTED** [review]
Pre-review, a couple of description suggestions.

File: features/transform-box.yml (hunk 2 above)
Comment (jamesnw) [reply features/transform-box.yml]: "Oddly phrased" seems to be the standard when looking at descriptions of this property. I updated it, and hope this is one standard I didn't match, but the phrasing is tricky.

File: features/transform-box.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: transform-box
+description: The `transform-box` CSS property sets the position and dimensions, or reference box, relative to which transformations are calculated.
```
Comment (captainbrosset) [inline features/transform-box.yml:2]:
```suggestion
description: The `transform-box` CSS property sets the position and dimensions of the reference box relative to which an element's individual transformations are calculated.
```

I was having trouble parsing the initial sentence. Maybe I misunderstood what you meant.
I added the word "individual" in there in an attempt to make it clear that this only applies for individual transform properties (if, indeed, this is how this works).

**Review by captainbrosset — APPROVED** [review]
Approving with a comment on the description. But it feels good enough for me not to have to block/review again.

File: features/transform-box.yml (hunk 3 above)
Comment (jamesnw) [reply features/transform-box.yml]: I liked the phrasing, but removed the word `individual`- it applies to the individual transform properties (scale, translate, rotate) as well as `transform`.

**Review by ddbeck — APPROVED** [review]
Happy enough to merge. Thank you!

**Pull request merged**